mitmproxy-rs/src/udp_client.rs

```diff
@@ -113,6 +113,14 @@ impl UdpClientTask {
             tokio::select! {
                 // wait for transport_events_tx channel capacity...
                 len = self.socket.recv(&mut udp_buf), if packet_tx.is_some() => {
+                    #[cfg(windows)]
+                    if let Err(e) = &len {
+                        if matches!(e.raw_os_error(), Some(10054)) {
```

Comment:
And there I thought "magic numbers bad" :) Is this a specific errno? Maybe it's defined (and named!) somewhere? At the very least, it should be a named `const`  so it's clear what this number means.

Reply:
Totally agree on adding a constant, also do you think it'd be better if we wrap this in a function somewhere (probably in `src/util.rs`) and call it from there? I'm not too experienced in rust so I'm not sure what's the best pattern here

Reply:
Looks good, thanks!